_episodes/05-writing-scripts.md

```diff
@@ -3,46 +3,91 @@ title: "Writing Scripts"
 teaching: 0
 exercises: 0
 questions:
-- How can we easily execute a commonly used set of commands.
+- How can we automate a commonly used set of commands?
 objectives:
-- Use the nano text editor to modify text files.
-- Write a basic shell script
-- Use the bash command to execute a shell script.
-- Use chmod to make a script an executable program.
+- Use the `nano` text editor to modify text files.
+- Write a basic shell script.
+- Use the `bash` command to execute a shell script.
+- Use `chmod` to make a script an executable program.
 keypoints:
-- Scripts are a collection of commands executed together
+- Scripts are a collection of commands executed together.
 ---
 
 ## Writing files
 
-We've been able to do a lot of work with files that already exist, but what if we want to write our own files. Obviously, we're not going to type in a FASTA file, but we'll see as we go through other tutorials, there area lot of reasons we'll want to write a file, or edit an existing file.
+We've been able to do a lot of work with files that already exist, but what if we want to write our own files. We're not going to type in a FASTA file, but we'll see as we go through other tutorials, there area lot of reasons we'll want to write a file, or edit an existing file.
 
-To write in files, we're going to use the program `nano`. We're going to create a file to take notes on what we've been doing with the data files in `~/dc_sample_data/untrimmed_fastq`.
+To add text to files, we're going to use a text editor called Nana. We're going to create a file to take notes about what we've been doing with the data files in `~/dc_sample_data/untrimmed_fastq`.
```

Comment:
I must admit that Nana does sound friendlier I would recommend to change it to Nano for easier recognition

Reply:
:-)